integrations/facebook-pixel/lib/index.js

```diff
@@ -198,15 +198,22 @@ FacebookPixel.prototype.productListViewed = function(track) {
   var self = this;
   var contentType;
   var contentIds = [];
+  var contents = [];
   var products = track.products();
   var customProperties = this.buildPayload(track, true);
 
   // First, check to see if a products array with productIds has been defined.
   if (Array.isArray(products)) {
     products.forEach(function(product) {
```

Comment:
I know this wasn't introduced in this PR, but could we change the `forEach` to a traditional `for...loop` for the products array?